src/dispatcher/mod.rs

```diff
@@ -137,6 +137,13 @@ impl Dispatcher {
                     name
                 )));
             }
+            // check if service is not yet up but started
+            if let Some(_) = service.url {
```

Comment:
This would affect all services up and starting up

Reply:
right. i dont want to be able to remove a service if its starting or up. i would want to do a `remove_service` only if the service is not up